### ISSUE TITLE
statusline error when performing an interactive search and replace

### ISSUE THREAD

**Issue by chebro** [NvChad/NvChad#351]
**Describe the bug**
performing an interactive search and replace operation throws statuline errors

**To Reproduce**

1. open some file.
2. enter command mode and run `:%s/old/new/gc`

it throws the following error

```
replace with the (y/n/a/q/l/^E/^Y)?                                                
/home/chaos/.config/nvim/lua/plugins/configs/statusline.lua:232: attempt to index a nil value                                                                         
E15: Invalid expression: luaeval('require("galaxyline").component_decorator')("left_round")                                                                           
/home/chaos/.config/nvim/lua/plugins/configs/statusline.lua:232: attempt to index a nil value function: builtin#18 /home/chaos/.config/nvim/lua/plugins/configs/statusline.lua:232: attempt to index a nil value                                         
E15: Invalid expression: luaeval('require("galaxyline").component_decorator')("viMode_icon")                                                                          
/home/chaos/.config/nvim/lua/plugins/configs/statusline.lua:232: attempt to index a nil value function: builtin#18 /home/chaos/.config/nvim/lua/plugins/configs/statusline.lua:232: attempt to index a nil value function: builtin#18 /home/chaos/.config/nvim/lua/plugins/configs/statusline.lua:232: attempt to index a nil value         
E15: Invalid expression: luaeval('require("galaxyline").component_decorator')("ViMode")               
```

**Expected behavior**
the command should execute successfully without the `statusline.lua` error

**Desktop (please complete the following information):**
 - arch linux
 - kitty
 - v0.6.0-dev+43-g02bf251bb

**Comment by G-Rowell** [comment]
I can confirm this bug

**Comment by siduck76** [comment]
yea can confirm

**Comment by G-Rowell** [comment]
Need all of our org to confirm before any action is taken right?

> yea can confirm



@chebro 
I'll take a good look at this tomorrow! (it's late here)

**Comment by siduck76** [comment]
@G-Rowell no , its a bug lol

**Comment by G-Rowell** [comment]
Lol, was just joking
Very tired and stressed

My bad, it was a poor attempt at sarcasm lol

**Comment by siduck76** [comment]
bruh no sarcasms in the issues :angry:  , get serious!

**Comment by siduck76** [comment]
lul no neovim + chad  = NvChad ( not a joke)

**Comment by G-Rowell** [comment]
Valid points on all sides,
let's take a moment to reflect and then come back with fresh minds?

Maybe if we make our own `Git` website we could then integrate a bot to detect sarcasm and flag it as such?

:fireworks: :heavy_plus_sign: :+1: :100: :medal_sports: :exclamation: :grin:

**Comment by ashincoder** [comment]
Guys is this the place to discuss sarcasm :laughing:

**Comment by Akianonymus** [comment]
@G-Rowell How about fixing the issue

**Comment by G-Rowell** [comment]
Looking into this

**Comment by G-Rowell** [comment]
Should be fixed when #362 is merged
